content/en/docs/tutorials/clusters/apparmor.md

```diff
@@ -178,7 +178,18 @@ k8s-apparmor-example-deny-write (enforce)
 First, we need to load the profile we want to use onto our nodes. The profile we'll use simply
 denies all file writes:
 
-{{< code language="text" file="deny-write.profile" >}}
+```shell
```

Comment:
To be honest, I didn't check the shortcode for `code`. It smells like a nit in the parsing logic for file name.
Since we already have `codenew` to replace `code` shortcode, I am not inclined to fix the `code` shortcode.

Reply:
@kbhawkey, @tengqm  I agree that deny-write.profile has so few lines that it doesn't need to be a separate file. But I was curious.

This compile error appears starting with Hugo v0.56.0, which had a lot of changes to the codebase.

In the Kubernetes website code, neither the _code_ nor _codenew_ shortcodes could find _deny-write.profile_, which was in the same directory as apparmor.md (I tried several different things). To compile without error in Hugo v0.56+, I created a new directory:
 ```content/en/examples/tutorials/clusters```

Then moved _deny-write.profile_ from  ```content/en/docs/tutorials/clusters``` to the new directory since _codenew_ looks for files in _examples_. 

Finally I changed _content/en/docs/tutorials/clusters/apparmor.md_ line 181 to use the _codenew_ shortcode:  
```{{< codenew language="text" file="tutorials/clusters/deny-write.profile" >}}```